### PR TITLE
update to load enrichment flow

### PR DESCRIPTION
**Pull request by ranum**
Script update to place enrichment flow on nifi canvas.  Also added better optional argument support for developers who want to load custom registries/buckets/versions instead of the default/latest. Note: nifi-setup:0.0.4 has already been built and pushed to dockerhub

### CODE REVIEW

File: clinical-ingestion/utilities/setupClinicalIngestionFlow.py [hunk 1]
```diff
@@ -40,128 +66,134 @@ def main():
...
-    if debug:
-        print(dict(resp.json()))
-    flowDict = dict(resp.json())
-    for flow in flowDict["versionedFlows"]:
+        #search for registry
+        regFound = False
+        for regs in respDict["registries"]:
+            if debug:
+                print(regs)
+            if regs["registry"]["name"] == flowConfig[aflow][0]:
```
Comment (ibmatc) [inline clinical-ingestion/utilities/setupClinicalIngestionFlow.py:98]: Nit: I would prefer to see "bucketName=flowConfig[aflow][0]" above, and then use that name, as this array has un-obvious content.

File: clinical-ingestion/utilities/setupClinicalIngestionFlow.py [hunk 2]
```diff
@@ -40,128 +66,134 @@ def main():
...
-        print("script failed-no matching flow found:", flowName)
-        exit(1)  #if we don't find the specific bucket then we are done
-
-    #found the flow so now go ahead and find the latest version
-    #unless latest is not None because then version already provided explicitly
-    if latest == None:
-        versionURL = flowURL + "/" + theFlow + "/" + "versions"
-        resp = requests.get(url=versionURL)
+                print(bucket)
+            if bucket["bucket"]["name"] == flowConfig[aflow][1]:
```
Comment (ibmatc) [inline clinical-ingestion/utilities/setupClinicalIngestionFlow.py:119]: same comment about using a variable

**Review by ibmatc — APPROVED** [review]
LGTM - two minor comments

File: clinical-ingestion/utilities/setupClinicalIngestionFlow.py (hunk 1 above)
Comment (ranum) [reply clinical-ingestion/utilities/setupClinicalIngestionFlow.py:98]: Yeah, I like that...I'll make the change in both places

**Pull request merged**